apps/web-portal/lib/appEndpoint.mjs

```diff
@@ -0,0 +1,18 @@
+import {makeRunner} from '@zorko-io/util-use-case'
+import express from 'express'
+
```

Comment:
I think that I do not get an idea of this mediator this looks very generic for me and configuration diff methods is too confusing. Maybe this can be more simple? It is just an endpoint list that will be as maximum 10-20 items

Reply:
Also, an express specific that endpoint declaration order matters, so I not sure how this function will handle this

Reply:
already removed